weather.py

```diff
@@ -2,116 +2,106 @@
 import sys
 import csv
 from datetime import datetime
+from pathlib import Path
 
-
-months =[]
+data_folder = Path("C:/Som/Personal Projects/weatherdata/")
```

Comment:
This will still not work on my local machine if I try to run.

Reply:
Yes, that's correct. i have to give a generic folder name only like weatherdata and pathlib is resolve it for all kind of OS i guess

Reply:
Did you also look into `os.path` https://docs.python.org/3/library/os.path.html